src/main/views/cop-daily-cause-list.njk

```diff
@@ -0,0 +1,139 @@
+        {% set sessionCount = sessionCount + 1 %}
+        {% for session in courtRoom['session'] %}
+          <div class="govuk-accordion__section govuk-accordion__section--expanded">
+            <div class="govuk-accordion__section-header">
+              <h2 class="govuk-accordion__section-heading">
+            <span class="govuk-accordion__section-button" id="accordion-default-heading-{{ sessionCount }}">
+              {{ beforeHon }} {{ session['formattedJudiciary'] }}
+            </span>
+              </h2>
+            </div>
+
+            <div id="accordion-default-content-{{ sessionCount }}" class="govuk-accordion__section-content"
+                 aria-labelledby="accordion-default-heading-{{ sessionCount }}">
+              <div class="parent-box overflow-table">
+                <table class="govuk-table">
+                  <thead class="govuk-table__head">
+                  <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header">{{ startTime }}</th>
+                    <th scope="col" class="govuk-table__header">{{ caseRef }}</th>
+                    <th scope="col" class="govuk-table__header">{{ caseDetails }}</th>
+                    <th scope="col" class="govuk-table__header">{{ hearingType }}</th>
+                    <th scope="col" class="govuk-table__header">{{ timeEstimate }}</th>
+                    <th scope="col" class="govuk-table__header">{{ hearingChannel }}</th>
+                  </tr>
+                  </thead>
+                  <tbody class="govuk-table__body">
+                  {% for sitting in session['sittings'] %}
+                    {% set durationAsHours = 0 %}
+                    {% if sitting['durationAsHours'] !== 0 %}
+                      {% if sitting['durationAsHours'] > 1 %}
+                        {% set durationAsHours = sitting['durationAsHours'] + ' ' + hearingDurationHours %}
+                        {% elif sitting['durationAsHours'] === 1 %}
+                        {% set durationAsHours = sitting['durationAsHours'] + ' ' + hearingDurationHour %}
+                      {% endif %}
+                    {% endif %}
+                    {% set durationAsMinutes = 0 %}
+                    {% if sitting['durationAsMinutes'] !== 0 %}
+                      {% if sitting['durationAsMinutes'] > 1 %}
+                        {% set durationAsMinutes = sitting['durationAsMinutes'] + ' ' + hearingDurationMinutes %}
+                        {% elif sitting['durationAsMinutes'] === 1 %}
+                        {% set durationAsMinutes = sitting['durationAsMinutes'] + ' ' + hearingDurationMinute %}
+                      {% endif %}
+                    {% endif %}
+                    {% set duration = '' %}
+                    {% if durationAsHours === 0 and durationAsMinutes === 0 %}
+                      {% set duration = '' %}
+                      {% elif durationAsHours !== 0 and durationAsMinutes !== 0 %}
```

Comment:
For Java code, the indentation for 'else if' will be the same as the one for 'if'. If that different for nunjunks where 'elif' has to be further indented?

Reply:
Should be the same, IDE auto formatted it wrong!